silta/silta.yml

```diff
@@ -65,5 +65,10 @@ silta-release:
 varnish:
   enabled: true
 
+redis:
+  enabled: false
```

Comment:
The build went through this time!

I wonder though, should this be set to `enabled: true`?

Reply:
@Jancis  ^

Reply:
for testing yes, but i doubt it's required for all out feature branches.
probably?
@Rade333 might have better opinion

Reply:
At least for the drupal k8 project it is set to false: 

https://github.com/wunderio/drupal-project-k8s/blob/fbffed4227e966bd0828cddabf362b72c112e127/silta/silta.yml#L16C1-L19C20

Would make sense here as well then, probably

Reply:
Let's keep it as false